lib/ask/audio.ex

```diff
@@ -17,7 +17,10 @@ defmodule Ask.Audio do
     timestamps()
   end
 
+  @valid_extensions ~w(.mp3 .wav)
+  @valid_mime_types ~w(audio/mpeg audio/wave audio/wav audio/x-wav audio/x-pn-wav)
```

Comment:
Do we care about these ones? As long as we get something that can get converted to a valid MP3, maybe we don't care what that something was?

Reply:
Yes, because there are mimetypes that `sox` doesn't support, and also because we probably don't want to support all possible audio formats.